### ISSUE TITLE
Password User.pw1 is no more valid

### ISSUE THREAD

**Issue by ibawwad** [MicrosoftLearning/MS-101T00-Microsoft-365-Mobility-and-Security#152]
# Module: 00
#### Step: 00

Description of issue
The password for suggested to be updated for users [User.pw1] is always rejected by MS password policy, it seems that it's becoming very restrictive.
The password is only accepted if you leave the option to (Require this user to change their password when they first sign in) other than this it's not accepted
Repro steps:

1.
1.
1.

**Comment by tfrink** [comment]
The current version seems very flaky, so I'm hesitant to make any changes to the instruction at this point. I have tested this extensively. Sometimes it accepts User.pw1 even when the "Require this user to change their pw" is not selected, and other times it says it's too weak. I've also gotten it to accept User.pw1 if I select the "Require this user to change their pw" option, select Next, then select Back, and then clear the "Require this user to change their pw" option. Then it accepts it and lets me click Next. Also, I've found that even when I clear the "Require this user to change their pw" option, any pw that I enter is rejected as too weak, even when I enter 10-20 character passwords with all sorts of characters (e.g. uuieQTye_23+3er, that kind of thing). It seems like everything I enter is considered too weak. So I'm guessing this logic will have to be fixed, so I don't want to make any changes at this point. Besides, as I mentioned, when it gets into this loop of rejecting every pw that I enter, I can't find a pw that is acceptable.
